src/locale/locales/np/messages.po

```diff
@@ -0,0 +1,9073 @@
+msgid ""
+"POT-Creation-Date: 2024-12-04 15:45+0300\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: np\n"
+"Project-Id-Version: Nepali localization for bluesky-social-app\n"
+"Report-Msgid-Bugs-To: Divyaswor Makai Shrestha <makaidivyaswor@gmail.com>\n"
+"PO-Revision-Date: \n"
+"Last-Translator: Divyaswor Makai Shrestha <makaidivyaswor@gmail.com>\n"
+"Language-Team: divyaswormakai\n"
+"Plural-Forms: \n"
+
+#: src/screens/Messages/components/ChatListItem.tsx:130
+msgid "(contains embedded content)"
+msgstr "सामग्री समावेश गरिएको छ"
+
+#: src/screens/Settings/AccountSettings.tsx:57
+#: src/view/com/modals/VerifyEmail.tsx:150
+msgid "(no email)"
+msgstr "इमेल छैन"
+
+#: src/view/com/notifications/FeedItem.tsx:232
+#: src/view/com/notifications/FeedItem.tsx:327
+#~ msgid "{0, plural, one {{formattedCount} other} other {{formattedCount} others}}"
+#~ msgstr "{0, plural, one {{formattedCount} अन्य} अन्य {{formattedCount} अन्यहरू}}"
+
+#: src/lib/hooks/useTimeAgo.ts:156
+msgid "{0, plural, one {# day} other {# days}}"
+msgstr "{0, plural, one {# दिन} अन्य {# दिनहरू}}"
```

Comment:
```suggestion
msgstr "{0, plural, one {# दिन} other {# दिनहरू}}"
```
The formatting of the strings involving plurals is tricky, both `one` and `other` need to remain in English as they are syntax and not translatable strings.

This also applies to quite a few other strings that follow ⬇️

Reply:
updated the scripts likewise